build.zig.zon

```diff
@@ -0,0 +1 @@
+.{ .name = "clap", .version = "0.7.0", .paths = .{""} }
```

Comment:
Please run `zig fmt` on this file. Also, pretty sure we need to populate `paths`. I saw some resource that documents the fields of `build.zig.zon` but I cannot find it right now.

Reply:
Actually, `zig fmt` removes the new lines. :(